src/components/AttachmentPicker/index.native.tsx

```diff
@@ -338,11 +301,12 @@ function AttachmentPicker({type, children, shouldHideCameraOption}) {
                     onCanceled.current();
                 }}
                 isVisible={isVisible}
-                anchorPosition={styles.createMenuPosition}
+                anchorRef={popoverRef}
+                // anchorPosition={styles.createMenuPosition}
```

Comment:
Let's remove both lines here (anchorRef is unused in the file, createMenuPosition doesn't exist)